src/ontology/cl-edit.owl

```diff
@@ -22344,9 +22349,9 @@ SubClassOf(obo:CL_0008004 obo:CL_0000187)
 
 # Class: obo:CL_0008005 (obliquely striated somatic muscle cell)
 
-AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:8720463") obo:IAO_0000115 obo:CL_0008005 "A somatic muscle cell that is oblquely striated.  Examples include the somatic muscles of nematodes.")
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:8720463") Annotation(oboInOwl:hasDbXref "doi:10.3908/wormatlas.1.7") Annotation(oboInOwl:hasDbXref "https://www.hh.um.es/pdf/Vol_11/11_1/Ultrastructure%20of%20invertebrate%20muscle%20cell%20types.pdf") obo:IAO_0000115 obo:CL_0008005 "A somatic muscle cell that is obliquely striated and mononucleated. Examples include the somatic muscles of nematodes.")
```

Comment:
Please, change the pdf link to doi? doi:10.14670/HH-11.181

Reply:
I have tried before, but it takes me to a page saying doi not found

Reply:
Then you can use either the PMID or the URL https://api.semanticscholar.org/CorpusID:8821317. Normally we would add the PMID but in this case it doesn't connect with the article. I would probably still use the PMID, but I leave to your criterium.